packages/react-components/react-dialog/src/components/DialogSurface/__snapshots__/DialogSurface.test.tsx.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DialogSurface renders a default state 1`] = `
+<div>
+  <div
+    class="fui-DialogSurface"
+    data-tabster="{\\"deloser\\":{},\\"modalizer\\":{\\"id\\":\\"modal-9\\",\\"isOthersAccessible\\":false}}"
```

Comment:
As it was mentioned, let's use `resetIdsForTests` to be sure that we don't have side-effects in snapshots